tests/scripts/create_subset_ebas.py

```diff
@@ -0,0 +1,288 @@
+
+    # directory containing NASA Ames files
+    FILEDIR_SRC = r_lowlev.file_dir
+    print(f"checking for invalid files in static file list...")
+    CURRENT_OK, files_valid, files_invalid = check_outdated(FILEDIR_SRC)
+    print(f"done...")
+
+    if not CURRENT_OK:
+        print(f"outdated files: {files_invalid}")
+        print("---------------------WARNING------------------------------")
+        print(
+            "Some files in current EBAS subset do not exist anymore in "
+            "the most recent EBAS database, consider updating them"
+        )
+        print("---------------------WARNING------------------------------")
+        raise Exception
+
+    if not OUTBASE.exists():
+        OUTBASE.mkdir()
+
+    if not FILES_DEST.exists():
+        FILES_DEST.mkdir()
+
+    if (
+        SEARCH_PROBLEM_FILES
+    ):  # use this block to find interesting files, e.g. that throw certain exceptions
+        var = "concpm10"
+        f0 = 1100
+        f1 = f0 + 1000
+
+        reader = pya.io.ReadEbas()
+        reader.read(var, first_file=f0, last_file=f1)
+        for i, file in enumerate(reader.files_failed):
+            try:
+                data = reader.read_file(file, var)
+            except Exception as e:
+                print(i)
+                print(os.path.basename(file))
+                print(e)
+                try:
+                    fd = pya.io.EbasNasaAmesFile(file)
+                    print(fd.station_name)
+                except Exception as e:
+                    print(e)
+
+    else:
+        infofile = JSON_FILE
+        if os.path.exists(infofile):
+            with open(infofile, "r") as f:
+                current_files = simplejson.load(f)
+        else:
+            current_files = {}
+
+        VARS_STATFIX = ["sc550dryaer", "ac550aer", "conco3", "concpm10"]
+        VARS_STATVAR_NUM = 2
+        VARS_STATVAR = ["concno2", "vmrno2", "vmro3", "concpm25"]
+
+        STATSFIX = ["Jungfraujoch", "Birkenes II", "*Kosetice*", "Troll"]
+
+        all_files = {}
+        overlap_found = False
+        for var in VARS_STATFIX + VARS_STATVAR:
+            if var in current_files and not UPDATE_EXISTING:
+                all_files[var] = current_files[var]
+                continue
+            print(var)
+            ropts = {}
+            if var.startswith("conc") or var.startswith("vmr"):
+                ropts["try_convert_vmr_conc"] = False
+            print(f"reading var {var}")
+            data = reader.read(NAME, var, **ropts)
+            print("reading done")
+            if var in VARS_STATFIX:
+                files, overlap_found = get_files_var_statlist(data, var, STATSFIX, overlap_found)
+            else:
+                files = get_files_var_statnum(data, var, VARS_STATVAR_NUM)
+            all_files[var] = files
+
+        if not UPDATE:
+            print("NOTHING WILL BE COPIED TO TEST DATA")
+        else:
+
+            src = Path(EBAS_BASE_DIR).joinpath("data")
```

Comment:
`EBAS_BASE_DIR` was upgraded to a `Path` object on a previous suggestion, so this libe becomes

``` python
            src = EBAS_BASE_DIR / "data"
```

Reply:
I put that into a separate issue #604. A bit out of scope for now.